### ISSUE TITLE
Possibility to set a different default timeout would be nice!

### ISSUE THREAD

**Issue by drahkrub** [joaonetocastro/vuetify-snackbar-toast#1]
Hi, very nice and small plugin - thank you! :-)

Would be great to be able to set a different **default** timeout - 7000ms are pretty long....

It can be overriden by providing `config` when calling the different functions on `$toast`, but changing the default timeout on setup would be great!

**Comment by joaonetocastro** [comment]
yes, in the next version i'll add the feature to change default timeout, and to customize the position too.
if you want i can help you develop the feature and merge the PR, it would be basically in the file toast/index.js making const defaultValues as a static prop for the Toast class, and passing a config object to the constructor, then putting it into the defaultValues prop for the class
